# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: 1c3926cd7004
# Full Hash: 1c3926cd7004e7cba4b2265caee686d6201d21d2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Move VideoConduit to call worker thread for mCall and related access. r=ng,bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112851
# ==============================================================================

diff -r 844625c1d96d -r 1c3926cd7004 dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Sun Apr 18 22:14:28 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Wed Apr 07 11:36:28 2021 +0200
@@ -331,6 +331,11 @@
    * Factory function to create and initialize a Video Conduit Session
    * @param  webrtc::Call instance shared by paired audio and video
    *         media conduits
+   * @param  aOptions are a number of options, typically from prefs, used to
+   *         configure the created VideoConduit.
+   * @param  aPCHandle is a string representing the RTCPeerConnection that is
+   *         creating this VideoConduit. This is used when reporting GMP plugin
+   *         crashes.
    * @result Concrete VideoSessionConduitObject or nullptr in the case
    *         of failure
    */